uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/calendar-feeds/services/apis/calendar-feed.api.js

```diff
@@ -21,7 +21,8 @@ define([
      * @return {Promise} resolved with an array of objects of feeds
      */
     function all () {
-      return this.sendGET('LeaveRequestCalendarFeedConfig', 'get', {}, false)
+      return this.sendGET('LeaveRequestCalendarFeedConfig', 'get',
+        { is_active: true })
```

Comment:
the PR description and previous code had `'LeaveRequestCalendarFeedConfig', 'get', {}, false` (false), but now we don't. Is this intentional? (Not event sure what this "false" means.

Reply:
Yes, intentional, it forces not saving cached api response. I don't think this is necessary, so I removed that. I updated the commend section.